### PR TITLE
Tutorial 4 pvmodule performance

### PR DESCRIPTION
**Pull request by mikofski**
Adds content for tutorial 4 on module performance.  Just wanted to get something in before tomorrow. It still needs some markdown.

### CODE REVIEW

**Comment by mikofski** [comment]
looks like you can view the goods here: https://github.com/mikofski/PVSC48-Python-Tutorial/blob/tutorial-4-pvmodule-performance/Tutorial%204%20-%20Model%20a%20Module's%20Performance.ipynb

**Comment by mikofski** [comment]
ok, @shirubana and @kanderso-nrel maybe you can take another look now?

https://nbviewer.jupyter.org/github/mikofski/PVSC48-Python-Tutorial/blob/tutorial-4-pvmodule-performance/Tutorial%204%20-%20Model%20a%20Module%27s%20Performance.ipynb
If it's okay with you, maybe you can make comments, or merge this, and make edits in a separate PR?

I will add the parameter generation in another PR okay? I'd like to merge this asap, so we're all working on the same set of notebooks, okay? 

thanks!

File: Tutorial 4 - Model a Module's Performance.ipynb [hunk 1]
```diff
@@ -4,25 +4,586 @@
...
+    "2. Calculate the single diode parameters for a set of PV module test conditions known as IEC61853.\n",
+    "3. Produce an IV curve for each of the IEC61853 test conditions.\n",
+    "4. Derive the CEC model parameters based on Isc, Voc, Imp, and Vmp CEC measurements.\n",
+    "\n",
+    "## STC\n",
+    "The most basic condition is called standard testing conditions or STC, which is considered the reference for most PV modules. For example, all of the PV modules in the SAM CEC module database list their nameplate power at STC.\n",
+    "* irradiance: 1000-W/m&sup2;\n",
+    "* cell temperature: 25&deg;C\n",
+    "* angle of incidence (AOI): 90&deg;\n",
+    "* sepctrum: AM1.5g (ASTM G-173)\n",
```
[the remark this answers is not in the document]
Comment (kanderso-nrel) [reply Tutorial 4 - Model a Module's Performance.ipynb:21]: Hopefully I am not brainfarting about 0 vs 90 AOI

File: Tutorial 4 - Model a Module's Performance.ipynb [hunk 2]
```diff
@@ -4,25 +4,586 @@
...
...anslate(162.084961 -15.625)scale(0.7)\" xlink:href=\"#DejaVuSans-Oblique-108\"/>\r\n      <use transform=\"translate(184.267578 0.78125)\" xlink:href=\"#DejaVuSans-32\"/>\r\n      <use transform=\"translate(216.054688 0.78125)\" xlink:href=\"#DejaVuSans-55\"/>\r\n      <use transform=\"translate(279.677734 0.78125)\" xlink:href=\"#DejaVuSans-53\"/>\r\n      <use transform=\"translate(343.300781 0.78125)\" xlink:href=\"#DejaVuSans-32\"/>\r\n      <use transform=\"translate(375.087891 0.78125)\" xlink:href=\"#DejaVuSans-Oblique-67\"/>\r\n     </g>\r\n    </g>\r\n   </g>\r\n  </g>\r\n </g>\r\n <defs>\r\n  <clipPath id=\"p01b3f1e129\">\r\n   <rect height=\"298.325625\" width=\"526.396875\" x=\"34.240625\" y=\"22.318125\"/>\r\n  </clipPath>\r\n  <clipPath id=\"pa9a56e605e\">\r\n   <rect height=\"298.325625\" width=\"526.396875\" x=\"604.840625\" y=\"22.318125\"/>\r\n  </clipPath>\r\n  <clipPath id=\"pe2ce091868\">\r\n   <rect height=\"298.325625\" width=\"526.396875\" x=\"34.240625\" y=\"376.918125\"/>\r\n  </clipPath>\r\n  <clipPath id=\"p23a4eac132\">\r\n   <rect height=\"298.325625\" width=\"526.396875\" x=\"604.840625\" y=\"376.918125\"/>\r\n  </clipPath>\r\n </defs>\r\n</svg>\r\n",
```
Comment (kanderso-nrel) [inline Tutorial 4 - Model a Module's Performance.ipynb:540]: Since when can you get SVGs from matplotlib?!  Very cool!

**Review by kanderso-nrel — APPROVED** [review]
This is great!  I left some minor proofreading edits below.  What do you think about having the pvlib docs links point to `stable` instead of `latest`?

File: Tutorial 4 - Model a Module's Performance.ipynb [hunk 3]
```diff
@@ -4,25 +4,586 @@
...
+     },
+     "metadata": {
+      "needs_background": "light"
+     }
+    }
+   ],
+   "source": [
+    "# plot the calculated curves:\n",
```
[the remark this answers is not in the document]
Comment (mikofski) [reply Tutorial 4 - Model a Module's Performance.ipynb:549]: I agree, I think it's a great idea, and I wanted to do it too. It's only b/c the 50&deg;Cand 75&deg;C plots have missing 100-W/m&sup2; and 200-W/m&sup2; lines that throws the colors off. I'll try to fix it, if you figure it out first, go for it, thx

File: Tutorial 4 - Model a Module's Performance.ipynb (hunk 2 above)
Comment (mikofski) [reply Tutorial 4 - Model a Module's Performance.ipynb:540]: looks way better than compressed png

File: Tutorial 4 - Model a Module's Performance.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (mikofski) [reply Tutorial 4 - Model a Module's Performance.ipynb:549]: is this any better?
iv-curve-info

I also fixed the order 25degC and 50degC were mixed up, check it out:
https://nbviewer.jupyter.org/github/mikofski/PVSC48-Python-Tutorial/blob/tutorial-4-pvmodule-performance/Tutorial%204%20-%20Model%20a%20Module%27s%20Performance.ipynb

if it's okay now, can you merge it, and I'll try to do the last part in another pr

**Comment by kanderso-nrel** [comment]
Thanks @mikofski for putting up with my plot bike shedding :)

**Pull request merged**